### ISSUE TITLE
[CLOSED] Hellfire Peninsula : Zeth'Gor : Aggro through the wall 

### ISSUE THREAD

**Issue by netherspite** [netherspite/cc-buglist#1527]
<a href="https://github.com/Mechaning"><img src="https://avatars2.githubusercontent.com/u/7338559?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [Mechaning](https://github.com/Mechaning)**    
_Thursday Aug 27, 2015 at 08:28 GMT_  
_Originally opened as https://github.com/ccshiro/cc-buglist/issues/1527_

----

*\* I saw that you closed a case on this issue but it's still problematic. **
https://github.com/ccshiro/cc-buglist/issues/927

Basically, the western palisade in Zeth'Gor doesn't act like a wall. Pretty much the same for the other structures in the area.  Got a nifty video for ye.
https://youtu.be/WvJxQ6bgm5o

**Comment by netherspite** [comment]
<a href="https://github.com/ccnim"><img src="https://avatars3.githubusercontent.com/u/10582080?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ccnim](https://github.com/ccnim)**  
_Thursday Aug 27, 2015 at 18:50 GMT_

----

It was closed as invalid because this happened and happens on retail.
